### PR TITLE
GEN-1301 - refact(Button): accept icon

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Button now accepts an icon that's rendered on the left. I use that for the buttons implemented by _ContactUs_ component.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3446** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3446" target="_blank">Graphite</a> 
    * **PR #3471** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3471" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3471?utm_source=stack-comment).

File: packages/ui/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ type CustomButtonProps = {
   variant?: 'primary' | 'primary-alt' | 'secondary' | 'secondary-alt' | 'ghost'
   size?: ButtonSize
   loading?: boolean
+  icon?: ReactNode
```
Comment (robinandeer) [inline packages/ui/src/components/Button/Button.tsx:18]: nit: I usually type these as PascalCase to indicate it's a react node

`Icon: ReactNode`

**Review by robinandeer — APPROVED** [review]
Looks cleaner indeed!

However, let's add this to Storybook story so we can keep track of all the use cases 😉 

Added via Giphy

**Review by gustaveen — APPROVED** [review]
Awesome! Me likey! 🥳

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Oct 24, 8:25 AM**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3471).

**Pull request merged**